extension/textext/__init__.py

```diff
@@ -338,10 +338,12 @@ def preview_callback(_text, _preamble, _preview_callback, _tex_command):
                                                     _tex_command)
 
                     with logger.debug("Run TexText GUI"):
-                        _, _, _, gui_wordwrap = asker.ask(save_callback, preview_callback)
+                        gui_config = asker.ask(save_callback, preview_callback)
 
                     with logger.debug("Saving global GUI settings"):
-                        self.config["gui_wordwrap"] = gui_wordwrap
+                        for key, value in gui_config.items():
```

Comment:
Let's store whole `gui_config` object as `gui`

```python
self.config["gui"] = gui_config
```

Reply:
I'm always amazed about what's possible... :smiley: I changed it in that way.